src/models/AccordionItem.php

```diff
@@ -16,19 +17,49 @@ class AccordionItem extends DataObject
     private static $db = [
         'Title'     => 'Varchar(255)',
         'Content'   => 'HTMLText',
+        'Group'     => 'Int(1)',
         'SortOrder' => 'Int',
     ];
 
     private static $has_one = [
         'Page' => 'Page'
     ];
 
+    /**
+     * @var array
+     */
+    private static $summary_fields = [
+        'Title' => 'Title',
+        'Group' => 'Accordion ID',
+    ];
+
     private static $default_sort = 'SortOrder ASC';
 
+    /**
+     * @return RequiredFields
+     */
+    public function getCMSValidator()
+    {
+        return new RequiredFields(['Group']);
+    }
+
+    /**
+     * @return FieldList
+     */
     public function getCMSFields()
     {
         $fields = parent::getCMSFields();
-        $fields->removeByName(['PageID', 'SortOrder']);
+
+        /** @var $group NumericField */
+        $group = $fields->fieldByName('Root.Main.Group');
+        $group->setDescription('All items that share the same accordion have to have the same accordion ID, accordion shortcode is [accordion,id=ID]');
```

Comment:
Just an ID is very unclear. At the least, give a list of the ID's currently set/available on this page.